### PR TITLE
Add shiny new badge to README.rst

### LINKED ISSUE #44: Beartype needs a badge
This is the most urgent issue in the backlog, _obviously_.

Consider this 100% hand-crafted, artisan, free-range, gluten free, non-GMO, certified organic, naturally essenced <sup>because "essence" [is a verb now](https://www.gocomics.com/calvinandhobbes/1993/01/25), apparently</sup> prototype made with love  and care using the [shields.io API](https://shields.io/#your-badge):

- [Bear-ified™](https://github.com/beartype/beartype)
  - `[Bear-ified™](https://github.com/beartype/beartype)`

_[insert preferred, "fixed it for ya", "hold my beer", or just plain "boom" meme here]_

### PR DESCRIPTION
**Pull request by posita**
Depends on ~beartype/beartype-assets#4~ _[merged](https://github.com/beartype/beartype-assets/pull/4#event-5102878481)_.
Step 2 of _n_ pursuant to beartype/beartype#44 (where _n_ is hopefully equal to 2).
In other words, aspirationally fixes #44.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -166,6 +166,42 @@ Let's install ``beartype`` with Anaconda_, because corporate standards are
    conda config --add channels conda-forge
    conda install beartype
 
+If you're feeling the love and wanting to celebrate, consider signaling that you're bear-ified:
+
+  YummySoft is now |bear-ified|!
+
+.. |bear-ified| image:: https://raw.githubusercontent.com/posita/beartype-assets/postita/0/add-bear-ified-badge/badge/bear-ified.svg
```
Comment (posita) [inline README.rst:173]: This needs to be changed to `https://raw.githubusercontent.com/beartype/beartype-assets/main/badge/bear-ified.svg`. I put this hear temporarily to enable rending until beartype/beartype-assets#4 is merged.

Comment (posita) [reply README.rst]: Should be fixed now.

**Comment by leycec** [comment]
Primavera! Bellissima! And other foreign superlatives I only pretend to understand! In short, I see your ~~Schwartz~~ reStructuredText is as big as mine. This is fantastic, because this is a work of @posita. It's in! We may now commence rejoicing by eating food and passing out.

Unrelatedly, is it just or me or did [code coverage just jump off the cliff without a wingsuit while no one was looking](https://app.codecov.io/gh/beartype/beartype)? I mean... **83% coverage.** Technically, that's still a positive integer. That's good. Unfortunately, at its current rate of monotonic decline, code coverage becomes a negative complex number in approximately three hours. That's bad.

**Comment by posita** [comment]
> Unrelatedly, is it just or me or did [code coverage just jump off the cliff without a wingsuit while no one was looking](https://app.codecov.io/gh/beartype/beartype)? I mean... **83% coverage.** Technically, that's still a positive integer. That's good. Unfortunately, at its current rate of monotonic decline, code coverage becomes a negative complex number in approximately three hours. That's bad.

Kind of like central banks’s interest rates! Clearly they are to blame. Although, looking at the commit for July 24 (where the drop happened), I wonder if [this comment](https://codecov.io/gh/beartype/beartype/commit/b4c79b97450bf11c9e951fd097a210657bda6e4b/#D8-133) provides any insight…?

**Comment by leycec** [comment]
> Kind of like central banks’s interest rates!

**It's a fine time to borrow, kids.**

A fine time to borrow!

> I wonder if this comment provides any insight…?

When you read "*gulp*" in a `git` commit, you just know you're in sharknado-infested waters. Luckily, it turned out the problem was that everything was broken. You'd think test coverage would fall farther than 83% when the codebase is taking on more water than a fishing boat trawling for *Jaws*. *You'd be wrong.*

> By the way, apologies if my repeated revisions iterating over a few text changes put unneeded stress on the CI system.

Oh! You're *sooooo* nice. I love apologies. They remind me of Japan. Most North Americans treat an apology like some emasculated sign of weakness akin to 16th-century anathema and self-flagellation by pig-greased burlap sack whips.

Of course, you have absolutely nothing to apologize for. You did phenomenal work here! I just chuckled when I scrolled for what felt like a subjective eternity without coming to the end of your forced pushes. We've all been there. I too have felt the late-night pain of "...just one more commit, honey, I swear." But why does she never believe me?

Thanks again for being so tremendous. If we ever bludgeon our politicos into giving us a Universal Basic Income (UBI) in Canada, the first thing I'll do is give half to you so that you can be liberated to be dirt poor too. :money_mouth_face:

**Pull request merged**